src/client/mapblock_mesh.cpp

```diff
@@ -52,6 +52,22 @@ void MeshMakeData::fillBlockData(const v3s16 &bp, MapNode *data)
 	m_vmanip.copyFrom(data, data_area, v3s16(0,0,0), blockpos_nodes, data_size);
 }
 
+void MeshMakeData::fillSingleNode(MapNode data)
+{
+	m_blockpos = {0, 0, 0};
+
+	m_vmanip.clear();
+	// area around 0,0,0 so that this positon has neighbors
+	const s16 sz = 4;
+	m_vmanip.addArea({v3s16(-sz), v3s16(sz - 1)});
+
+	u32 count = m_vmanip.m_area.getVolume();
+	for (u32 i = 0; i < count; i++) {
+		m_vmanip.m_data[i] = data;
```

Comment:
Suggestion: Set all to air, but the middle. And remove `renderSingle()`.

Reply:
Good idea, will do.